buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -311,16 +371,19 @@ private void waitForProcessToExit(ProcessHandle processHandle) {
         }
     }
 
-    private void configure()  {
-        try {
-            Files.createDirectories(configFile.getParent());
-            Files.createDirectories(confPathRepo);
-            Files.createDirectories(confPathData);
-            Files.createDirectories(confPathLogs);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
+    private void createWorkingDir(Path distroExtractDir) throws IOException {
+        services.sync(spec -> {
+            spec.from(distroExtractDir.toFile());
```

Comment:
In ClusterFormationTasks, we always have a fresh directory we extract into. This isn't great for build performance, but it means we don't have any possiblity of old stuff being used. From a reproducibility perspective, I think we should wipe the data/logs/temp dirs here?

Reply:
FYI, `sync()` does exactly this. In contrast to `copy()` a sync first deletes the target folder before performing the copy.